bot/exts/moderation/infraction/infractions.py

```diff
@@ -480,6 +474,47 @@ async def apply_ban(
         async def action() -> None:
             await ctx.guild.ban(user, reason=reason, delete_message_days=purge_days)
 
+        if isinstance(user, Member) and any(role.id in constants.STAFF_PARTNERS_COMMUNITY_ROLES for role in user.roles):
+            confirmation_view = _utils.StaffBanConfirmationView(
+                allowed_users=(ctx.author.id,),
```

Comment:
having this allowed_users line is likely superfluous, since only MODERATION_ROLES can run this command inthe first place.

Reply:
Seems like `allowed_users` is a require argument, so I don't think any harm can come of it by just leave it